packages/firestore/test/unit/local/query_cache.test.ts

```diff
@@ -98,11 +89,17 @@ function genericQueryCacheTests(): void {
     );
   }
 
+  let persistence: Persistence;
   beforeEach(async () => {
+    persistence = await persistencePromise();
     cache = new TestQueryCache(persistence, persistence.getQueryCache());
     await cache.start();
   });
 
+  afterEach(async () => {
+    persistence.shutdown(/* deletaData= */ true);
```

Comment:
Actually, since it looks like you're going to need to merge master and let travis re-run anyway, I'll point out that you missed the second typo in this line.  delet_e_Data

Reply:
This was apparently a load-bearing typo. CI has been failing since I fixed it...